app/src/nxt/java/com/pileproject/drive/execution/NxtController.java

```diff
@@ -194,31 +197,32 @@ public void move(MotorDir leftMotorDir, MotorDir rightMotorDir) {
     }
 
     /**
-     * set motor's power as percent
+     * A setter of motor power.
      *
      * @param kind
-     * @param percent
+     * @param power
      */
-    public void setMotorPower(MotorKind kind, double percent) {
-        if (percent > 100.0) {
-            percent = 100.0;
-        } else if (percent < 0) {
-            percent = 0.0;
+    public void setMotorPower(MotorKind kind, int power) {
+        if (power > 100) {
+            power = 100;
+        } else if (power < 0) {
+            power = 0;
         }
 
         switch (kind) {
             case LeftMotor:
-                mLeftMotorPower = (int) ((MAX_MOTOR_POWER * percent) / 100.0);
+                mLeftMotorPower = power;
                 break;
             case RightMotor:
-                mRightMotorPower = (int) ((MAX_MOTOR_POWER * percent) / 100.0);
+                mRightMotorPower = power;
                 break;
         }
     }
 
     @Override
     public void finalize() throws RuntimeException {
```

Comment:
I see. To make matters worse, this doesn't override `Object`'s finalize (it is called explicitly by an Activity...). I'll fix it in other PR. Thanks!